sonic-utilities-tests/sku_create_test.py

```diff
@@ -0,0 +1,76 @@
+import sys
+import os
+import pytest
+import swsssdk
+import subprocess
+import filecmp
+import re
+import time
+
+test_path = os.path.dirname(os.path.abspath(__file__))
+modules_path = os.path.dirname(test_path)
+scripts_path = os.path.join(modules_path, "scripts")
+input_path = os.path.join(modules_path, "sonic-utilities-tests/sku_create_input")
+sku_def_file = os.path.join(input_path, "Mellanox-SN2700-D48C8.xml")
```

Comment:
@madhanmellanox looks like this file is not a part of sonic-utilities repo. How do you plan to distribute it?

Reply:
sku_create_test.py script and sku_create_input folder will be added to sonic-utilities repo as new file and new folder.